extensions/machine-learning-services/src/controllers/mainController.ts

```diff
@@ -110,13 +111,16 @@ export default class MainController implements vscode.Disposable {
 		let modelManagementController = new ModelManagementController(this._apiWrapper, this._rootPath,
 			azureModelsService, registeredModelService, predictService);
 
+		let dashboardWidget = new DashboardWidget(this._apiWrapper, this._rootPath);
+		dashboardWidget.register();
```

Comment:
this is using the new dashboard flow right? (looks so from the snapshot but just to confirm) did you have to do somehting extra there?

Reply:
no this registers the widget to show up in dashboard